lib/lists.nix

```diff
@@ -6,8 +6,8 @@ rec {
 
   inherit (builtins) head tail length isList elemAt concatLists filter elem genList;
 
-  /*  Create a list consisting of a single element.  `singleton x' is
-      sometimes more convenient with respect to indentation than `[x]'
+  /*  Create a list consisting of a single element.  `singleton x` is
```

Comment:
If it doesn't matter why are you changing it ? :)

I would also prefer the Markdown notation but it needs to be consistently applied, or at least have some sort of consensus that this is the notation that we want to use. Most of the lib/*.nix files are using that weird quoting syntax.